src/recorder.js

```diff
@@ -24,6 +24,7 @@ var Recorder = function( config ){
   this.config.encoderApplication = config.encoderApplication || 2049;
   this.config.encoderFrameSize = config.encoderFrameSize || 20;
   this.config.resampleQuality = config.resampleQuality || 3;
+  this.config.encoderComplexity = config.encoderComplexity || 9;
```

Comment:
Would prefer if a default was not specified so that we use the opus defined value.